sympy/sets/sets.py

```diff
@@ -1877,10 +1877,10 @@ def _union(self, other):
 
     def _contains(self, other):
         """
-        Tests whether an element, other, is in the set.
+        Tests whether an element 'other' is in the set.
 
         Relies on Python's set class. This tests for object equality
-        All inputs are sympified
+        All inputs are simplified
```

Comment:
I know simpify() function is there in sympy and with that reference it was written sympified . But my arguments are as follows:-

1. There is no such word like sympified in dictionary .
2. For the novice it would be better to understand if we keep it more simplified.

In case my points are not valid then i can revert that to sympified.